src/Microsoft.AspNetCore.WebUtilities/Properties/AssemblyInfo.cs

```diff
@@ -3,9 +3,11 @@
 
 using System.Reflection;
 using System.Resources;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: NeutralResourcesLanguage("en-us")]
```

Comment:
Sure, but seems unnecessary to create a diff? We generally don't have this sorted elsewhere.